trip_app/core/utils.py

```diff
@@ -11,3 +17,13 @@ def raise_for_status(resp):
 
     if http_error_msg:
         raise Exception(http_error_msg)
+
+
+def str_to_geopoint(data):
+    """ Convert string coordinates in Point object. E.g. '23.4 23.5'-> Point(23.4, 23.5) """
+    lon, lat = re.findall(r"[-+]?\d*\.?\d+|\d+", data)
```

Comment:
what are possibilities for the incoming `data`? I wonder why we need to use `re` here.

Reply:
Well... actually I choose over using `re` here to expand the range of incoming formats of data...
I expect the client would send us coordinates in string formats such as `'23.4 23.5'`, `'23.4, 23.5'`, `'[23.4 23.5]'`, `'(23.4 23.5)'`, `'[23,4, 23,5]'`.. We can make a rule here e.g. to accept only string with parenthesis consisting of 2 numbers separated by space, and essentially avoid using re. But I believe that such approach would make a life of clients easier.

Reply:
I see now. I'd cover it with some inline (doctests) tests then. 
Also, not sure if we want to parse `23,5`. Dot-separator shoud be enough.

Reply:
Test along with docstrings to this function should be added now. And yeah... comma separator is not easy to recognize therefore we'd like to accept only dot-separator coordinates. Thanks!